src/langrila/__init__.py

```diff
@@ -9,6 +9,7 @@
 )
 from .embedding.openai import OpenAIEmbeddingModule
 from .memory.json import JSONConversationMemory
+from .memory.cosmos import CosmosConversationMemory
```

Comment:
It could be better not to import in __init__.py because additional dependencies are needed.

Reply:
Got it. It should be moved to each Python scripts to use Cosmos.

Reply:
Yes, but it's OK.
I forgot to suggest.

```suggestion
```